README.md

```diff
@@ -1,7 +1,22 @@
-Autodesk VRED integration with Shotgun
-
+[![Build Status](https://dev.azure.com/shotgun-ecosystem/Toolkit/_apis/build/status/Engines/tk-vred?branchName=master)](https://dev.azure.com/shotgun-ecosystem/Toolkit/_build/latest?definitionId=56&branchName=master)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+## Documentation
 
-[![Build Status](https://dev.azure.com/shotgun-ecosystem/Toolkit/_apis/build/status/Engines/tk-vred?branchName=master)](https://dev.azure.com/shotgun-ecosystem/Toolkit/_build/latest?definitionId=56&branchName=master)
+This repository is a part of the Shotgun Pipeline Toolkit.
+
+- For more information about this app and for release notes, *see the wiki section*.
+- For general information and documentation, click here: https://support.shotgunsoftware.com/entries/95441257
+- For information about Shotgun in general, click here: http://www.shotgunsoftware.com/toolkit
+
+## Using this engine in your Setup
+
+All the engines that are part of our standard engine suite are pushed to our engine Store.
```

Comment:
"All the engines that are part of our standard configurations are pushed to our App Store."

Reply:
done